### ISSUE TITLE
Client HTML dropdown not displaying correctly on Firefox

### ISSUE THREAD

**Issue by ncclementi** [dask/distributed#5193]
In Firefox the arrow dropdowns on the HTML repr do not display, although I can still click and get the dropdown.

![dropdown](https://user-images.githubusercontent.com/7526622/128774018-810258a1-11d2-45fa-83b0-72b0a19dc594.gif)

Distributed version 2021.07.2 (tried on main too)
Browser Firefox 90.0.2 

cc: @jacobtomlinson

**Comment by jacobtomlinson** [comment]
I think @GenevieveBuckley also noticed this. I have been unable to reproduce.

We are just using HTML `<details>` elements which provide this functionality. Looking at the [MDN docs page](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) the screenshots there seem to show the arrow as expected.

![image](https://user-images.githubusercontent.com/1610850/128886677-5a5e78de-b7ac-4abe-bd1d-fb17c6062821.png)

I wonder if there is a browser bug or maybe an extension which is causing the triangle not to show.

**Comment by jacobtomlinson** [comment]
Oh I've managed to reproduce in Jupyter Classic! I generally only use Jupyter Lab.

My guess is that there is some CSS in Jupyter Classic which hides these triangles.

<img width="1282" alt="image" src="https://user-images.githubusercontent.com/1610850/128887154-c42b94f0-d846-4871-b1f2-25096a1967c5.png">

**Comment by mrocklin** [comment]
cc @ian-r-rose

**Comment by ncclementi** [comment]
@jacobtomlinson Yes, I forgot to mention that I was on Jupyter classic. I don't have the problem on Jupiter lab.

**Comment by GenevieveBuckley** [comment]
Yeah, I noticed this too but figured it was more a difference in how things are displayed by default rather than a bug.

It does make it a little harder for people to notice that there are expandable details in there, so if it was easy to add that might be good
